index.html

```diff
@@ -14,6 +16,86 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+    <section class="intro">
+      <div class="overlay"></div>
+      <header class="header">
+        <nav class="nav-links">
+          <a href="#">home</a>
+          <a href="#">services</a>
+          <a href="#">blog</a>
+          <a href="#">contact</a>
+        </nav>
+        <div class="intro-text">
+          <h1>Hi I'm Kelechi Ogbonna</h1>
+          <p>Full Stack Web Developer</p>
+        </div>
+      </header>
+    </section>
+
+
+    <section class="personal-info">
+      <div class="about-me">
+        <h3>About Me</h3>
+        <p>
+          As a person who enjoys looking for solutions to problems, who sees challenges as a way to grow,
+          it's easy to see why I fell in love with programming. Being given the oppurturnity to help people solve their
+          problems
+          through building softwares has been my life goal. I enjoy using html and css to build beautiful web page
+          design
+          and
+          javascript to add magic to the page, while delivery simple and beautiful work.
+        </p>
+      </div>
+      <div class="about-me-img">
+        <img src="image/kell.jpg" alt="" srcset="">
+      </div>
+    </section>
+
+    <section class="work-info">
+      <div class="skills">
+          <h3>My Skills</h3>
+          <ul>
+            <li>HTML</li>
+            <li>CSS/LESS</li>
+            <li>Javascript</li>
+            <li>PostgreSQL</li>
+            <li>NodeJS</li>
+          </ul>
+      </div>
+    <div class="work-history">
```

Comment:
Be careful here. This div it's opening here but it's not closing.

To expand on this, your indentation is not always perfect, and that probably don't help you out to see when something like this happen. You could want to use VScode autoformat tool (right click on your page -> format document) or install some extensions like prettier.

Reply:
I have prettier installed. I'll try to use it more often